### PR TITLE
Remove emulate-hash, trace-hash and trace-next-block

### PR DESCRIPTION
**Pull request by anton-lisanin**
Delete everything related to accessing indexer database from the neon-cli/api

### CODE REVIEW

**Review by afalaleev — APPROVED** [review]

File: evm_loader/lib/src/types/indexer_db.rs [hunk 1]
```diff
@@ -1,176 +0,0 @@
...
-    "from_addr, COALESCE(to_addr, contract), calldata, value, gas_limit, nonce";
-
-impl IndexerDb {
-    pub async fn new(config: &ChDbConfig) -> Self {
-        let client = do_connect(
-            &config.indexer_host,
-            &config.indexer_port,
-            &config.indexer_database,
-            &config.indexer_user,
-            &config.indexer_password,
```
Comment (andreisilviudragnea) [inline evm_loader/lib/src/types/indexer_db.rs:28]: These fields can also be deleted from struct `ChDbConfig`.

Their corresponding env vars (`NEON_DB_INDEXER_HOST`, `NEON_DB_INDEXER_PORT`, `NEON_DB_INDEXER_DATABASE`, `NEON_DB_INDEXER_USER`, `NEON_DB_INDEXER_PASSWORD`) can be deleted too.

Also the values provided for these env vars from `docker-compose-ci.yml` and `INDEXER_DB_PASSWORD: ${{secrets.INDEXER_DB_PASSWORD}}` from `pipeline.yml` can be deleted too.

File: evm_loader/lib/Cargo.toml [hunk 2]
```diff
@@ -29,7 +29,6 @@ goblin = { version = "0.6.0" }
 scroll = "0.11.0"
 tokio = { version = "1", features = ["full"] }
 postgres = { version = "0.19", features = ["with-chrono-0_4", "array-impls"] }
```
Comment (andreisilviudragnea) [inline evm_loader/lib/Cargo.toml:31]: This can be deleted too.

File: evm_loader/lib/src/types/indexer_db.rs (hunk 1 above)
Comment (anton-lisanin) [reply evm_loader/lib/src/types/indexer_db.rs:28]: Deleted

File: evm_loader/lib/Cargo.toml (hunk 2 above)
Comment (anton-lisanin) [reply evm_loader/lib/Cargo.toml]: Deleted

**Review by andreisilviudragnea — APPROVED** [review]
Where will these APIs be moved after this PR?

**Comment by anton-lisanin** [comment]
They are completly removed. This should not have been there in a first place.

**Pull request merged**